usb_protocol/types/descriptors/uac2.py

```diff
@@ -633,9 +633,9 @@ class SidebandProtocols(IntEnum):
     "bCSourceID"          / DescriptorField(description="ID of the clock which is connected to this terminal"),
     "bNrChannels"         / DescriptorField(description="number of logical output channels in the terminal’s output channel cluster"),
     "bmChannelConfig"     / DescriptorField(description="describes the spatial location of the logical channels", default=0, length=4),
-    "bmControls"          / DescriptorField(description="OR combination of  ClockFrequencyControl, CopyProtectControl, ConnectorControl, ClusterControl, UnderflowControl and OverflowControl", default=0, length=2),
     "iChannelNames"       / DescriptorField(description="string descriptor index of the first logical channel name", default=0),
-    "iTerminal"           / DescriptorField(description="ID of the input terminal string description", default=0)
+    "bmControls"          / DescriptorField(description="OR combination of  ClockFrequencyControl, CopyProtectControl, ConnectorControl, ClusterControl, UnderflowControl and OverflowControl", default=0, length=2),
```

Comment:
Not your fault, but I think this description is incorrect? I think it should have OverloadControl and not ClockFrequencyControl.